frontend/javascripts/users/rooms/review_requests/insert_pull_info.js

```diff
@@ -0,0 +1,20 @@
+export default class InsertPullInfo {
```

Comment:
__[ESLint]__

Error - Parsing error: The keyword 'export' is reserved


<!-- Comment by [Sider](https://sider.review) -->